extension/test/newman/collection.json

```diff
@@ -1530,6 +1530,1444 @@
 					"_postman_isSubFolder": true
 				}
 			]
+		},
+		{
+			"name": "Sync",
```

Comment:
the test structure is off, I mean, we have "guest", "customer" and "sync". I feel like something went horribly wrong with this structure ^_^ @andre-kraus , can you help? In the last fav review my suggestion was to break it down into: 
```
-> get/add/delete
-> update
-> sync
```
I think it is better, no?